source/frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -38,6 +38,13 @@ export const ownershipFilterOptions: MultiSelectOption[] = [
   { id: 'isRetired', text: 'Retired' },
 ];
 
+export const tenureCleanupFilterOptions: MultiSelectOption[] = [
```

Comment:
The pattern for these is normally to get the list of code values from the backend and add those code values as a new lookup to the lookup codes.